web/src/components/Contact.jsx

```diff
@@ -0,0 +1,42 @@
+import youtube from '/svgs/youtube.svg'
+import instagram from '/svgs/instagram.svg'
+import linkedin from '/svgs/linkedin.svg'
+import twitter from '/svgs/twitterx.svg'
+
+function Contact(){
+    //TODO add SVG to each link
+    return (
+        <div className="contactContainer">
+            <div className="infoL">
+            <p>Laboratorio de Computación:</p>
+
+            <p>Correo: lc.fcefyn@gmail.com</p>
```

Comment:
Can we use an image/icon for GMAIL. For example https://upload.wikimedia.org/wikipedia/commons/7/7e/Gmail_icon_%282020%29.svg